package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -470,7 +470,8 @@ class DumpReader(base.ReaderBase):
     """
     format = 'LAMMPSDUMP'
     _conventions = ["auto", "unscaled", "scaled", "unwrapped",
-                    "scaled_unwrapped"]
+                    "charge", "scaled_unwrapped"]
```

Comment:
This is primarily from just for the coordinate conventions. For arbitrary columns I would add an `extra_keys` kwarg or something to `__init__`